lsp/nls/src/requests/completion.rs

```diff
@@ -780,6 +792,48 @@ pub fn handle_completion(
     Ok(())
 }
 
+fn handle_import_completion(
+    import: &OsString,
+    params: &CompletionParams,
+) -> io::Result<Vec<CompletionItem>> {
+    debug!("handle import completion");
+
+    let current_file = params
+        .text_document_position
+        .text_document
+        .uri
+        .to_file_path()
+        .unwrap()
+        .canonicalize()?;
```

Comment:
We've been using `nickel_lang_core::cache::normalize_path` instead of `canonicalize` because it behaves better for files that don't exist on the filesystem (which I think could happen here if they're editing a new file that hasn't yet been saved).